api/client/service/opts.go

```diff
@@ -462,7 +463,6 @@ func (opts *serviceOptions) ToService() (swarm.ServiceSpec, error) {
 			Delay:         opts.update.delay,
 			FailureAction: opts.update.onFailure,
 		},
-		Networks:     convertNetworks(opts.networks),
```

Comment:
Great. Thanks for the clarification. Just to confirm , if the manager is 1.13 and some of the workers are 1.12.1, then the only limitation would be that the `docker run` will not work in those 1.12.1 workers. The 1.13 workers and managers can enjoy this feature. correct ?